### ISSUE TITLE
Very slow tests due to encryption token generation

### ISSUE THREAD

**Issue by imsodin** [syncthing/syncthing#7344]
Tests locally time out every now and then, and the culprit is the test that checks that the handling of encryption tokens in the cluster config is correct. That's not so surprising, as it has to generate a lot of tokens with scrypt, which is designed to be slow.

I see two options: Caching generated tokens in the model or skipping this test when running with `-short`. On the one hand I would like to run the tests, on the other hand caching isn't very beneficial in production as we don't get cluster configs for the same device very often.

**Comment by calmh** [comment]
`TestCcCheckEncryption`? 12s on the build server, definitely `-short` material I think.

**Comment by imsodin** [comment]
Yes.

And `TestRequestReceiveEncryptedLocalNoSend` is slow too.

tl;dr of the stuff below: Scrypt is very good at being slow without using much cpu, and that confused me :D 

There it's a single password token. Sometimes the function takes ~2s, often 6-18s. The cpu-profile shows ~300ms spent in `scrypt`. Adding some "trace" logging around the scrypt step:

```golang
         .          .    474:   l.Infoln(25)
         .      280ms    475:   bs, err := scrypt.Key([]byte(password), knownBytes(folderID), 32768, 8, 1, keySize)
         .          .    476:   l.Infoln(27)
```

and the corresponding stdout:

```
2021/02/08 16:02:33.460326 encryption.go:474: INFO: 25
2021/02/08 16:02:41.673153 encryption.go:476: INFO: 27
```

I'll disable both when `-short`
